helmfile/charts/networkpolicy-service/templates/rook-ceph/mgr.yaml

```diff
@@ -0,0 +1,119 @@
+{{ if .Values.rookCeph.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-rook-ceph-mgr
+  namespace: rook-ceph
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app: rook-ceph-mgr
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              app: rook-ceph-operator
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels:
+              app: rook-ceph-tools
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels:
+              app: rook-ceph-mon
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels:
+              app: rook-ceph-osd
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+    - from:
+        {{- range $ip := .Values.global.scApiserver.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+    - from:
+        {{- range $ip := .Values.global.scNodes.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels:
+              app: csi-rbdplugin-provisioner
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+        - port: 9283
+          protocol: TCP
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              app: rook-ceph-osd
+      ports:
+        - port: 6800
+          protocol: TCP
+          endPort: 7300
+    - to:
+        {{- range $ip := .Values.global.scApiserver.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      ports:
+        - port: 6443
+          protocol: TCP
+    - to:
+        - podSelector:
+            matchLabels:
+              app: rook-ceph-operator
```

Comment:
Do you have any specific ports here it uses?

Reply:
@Xartos Thanks !! As discussed today , i have done the required changes !!